Rakefile

```
@@ -28,7 +28,7 @@ end
 
 task :travis do
   Rake::Task['yard'].invoke if !Beaker::Shared::Semvar.version_is_less(RUBY_VERSION, '2.0.0')
```

Since Ruby >= 2.1.8 is required (https://github.com/puppetlabs/beaker/blob/8ef9afb3690e43be5a38bfe3ef9f4502af6a4881/beaker.gemspec#L21), this can be simplified:
```suggestion
  Rake::Task['yard'].invoke
```

That also suggests you can use dependencies (untested).
```ruby
task :travis => [:yard, :test]
```